### ISSUE TITLE
AIX ncpa error log

### ISSUE THREAD

**Issue by metabsd** [NagiosEnterprises/ncpa#370]
I try to use API get all the stuff the api see and I got that error in ncpa_listener

```
2017-08-22 15:55:00,968 21430858 ERROR __new__() takes exactly 6 arguments (5 given)
Traceback (most recent call last):
  File "/tmp/test/ncpa/agent/listener/nodes.py", line 64, in walk
  File "/tmp/test/ncpa/agent/listener/nodes.py", line 148, in walk
  File "/tmp/test/ncpa/agent/listener/psapi.py", line 176, in <lambda>
  File "/tmp/test/psutil/psutil/__init__.py", line 2347, in users
  File "/tmp/test/psutil/psutil/_psaix.py", line 167, in users
TypeError: __new__() takes exactly 6 arguments (5 given)
2017-08-22 15:55:00,970 21430858 ERROR __new__() takes exactly 6 arguments (5 given)
Traceback (most recent call last):
  File "/tmp/test/ncpa/agent/listener/nodes.py", line 64, in walk
  File "/tmp/test/ncpa/agent/listener/nodes.py", line 148, in walk
  File "/tmp/test/ncpa/agent/listener/psapi.py", line 177, in <lambda>
  File "/tmp/test/psutil/psutil/__init__.py", line 2347, in users
  File "/tmp/test/psutil/psutil/_psaix.py", line 167, in users
TypeError: __new__() takes exactly 6 arguments (5 given)
```
After that query. NCPA work but the web interface not working for couple minutes and work again.

Wierd.

Thank you!!

**Comment by jomann09** [comment]
I will double check that this is what I think it is. I think that the AIX user information from psutil is not filled out properly. I'll definitely double check this and see if I can't fix it.

**Comment by jomann09** [comment]
I've fixed this ... it required a patch to the psutils version we are running on the AIX servers. I am going to have to validate this still works when we upgrade psutils although I did mention to the author of the port that this was an issue and what I did to resolve it.

This will be fixed in the 2.0.5+ builds.
